src/api/app/models/concerns/workflow_run_github_payload.rb

```diff
@@ -25,6 +25,10 @@ def github_target_repository_full_name
     payload.dig(:repository, :full_name) # for a push event
   end
 
+  def github_pr_number
+    payload[:number]
```

Comment:
`payload[:number]` does not exist anymore

Reply:
not? thats how we retrieved it before and it should be available through `payload[:number]`